### PR TITLE
Add django-bakery

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/django-bakery/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (isuruf) [inline recipes/django-bakery/meta.yaml]: Please add `noarch: python`

Comment (sannykr) [reply recipes/django-bakery/meta.yaml]: Thanks. I will make it noarch and re-render.

**Comment by sannykr** [comment]
@jakirkham : Could you please review and merge if everything looks okay.

**Comment by sannykr** [comment]
@synapticarbors : Could you please review and merge.

**Pull request merged**